### ISSUE TITLE
Error while installing through pip

### ISSUE THREAD

**Issue by pxl-th** [eosmoto/eosiopy#7]
While trying to install through pip I get this error:

Cause:
```
eosiopy/uECC.c:3:18: fatal error: uECC.h: No such file or directory
```

Complete log:
```
Running setup.py install for eosiopy ... error
    Complete output from command /usr/local/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-96azbit_/eosiopy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-xp1k3suz-record/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.6
    creating build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/eosioparams.py -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/sign.py -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/eosioapi.py -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/rawinputparams.py -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/config.py -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/exception.py -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/__init__.py -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/log.py -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/packedtransaction.py -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/utils.py -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/nodenetwork.py -> build/lib.linux-x86_64-3.6/eosiopy
    running egg_info
    creating eosiopy.egg-info
    writing eosiopy.egg-info/PKG-INFO
    writing dependency_links to eosiopy.egg-info/dependency_links.txt
    writing requirements to eosiopy.egg-info/requires.txt
    writing top-level names to eosiopy.egg-info/top_level.txt
    writing manifest file 'eosiopy.egg-info/SOURCES.txt'
    warning: manifest_maker: standard file '-c' not found
    
    reading manifest file 'eosiopy.egg-info/SOURCES.txt'
    writing manifest file 'eosiopy.egg-info/SOURCES.txt'
    copying eosiopy/uECC.c -> build/lib.linux-x86_64-3.6/eosiopy
    copying eosiopy/uECC.so -> build/lib.linux-x86_64-3.6/eosiopy
    running build_ext
    building 'uECC' extension
    creating build/temp.linux-x86_64-3.6
    creating build/temp.linux-x86_64-3.6/eosiopy
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/local/include/python3.6m -c eosiopy/uECC.c -o build/temp.linux-x86_64-3.6/eosiopy/uECC.o
    eosiopy/uECC.c:3:18: fatal error: uECC.h: No such file or directory
     #include "uECC.h"
                      ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/usr/local/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-96azbit_/eosiopy/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-xp1k3suz-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-build-96azbit_/eosiopy/
```

**Comment by eosmoto** [comment]
What system environment？

**Comment by eosmoto** [comment]
I have already released 0.1.3
